src/main/java/org/springframework/data/elasticsearch/client/util/RequestConverters.java

```diff
@@ -540,8 +540,10 @@ private static Request prepareReindexRequest(ReindexRequest reindexRequest, bool
 	public static Request updateByQuery(UpdateByQueryRequest updateByQueryRequest) {
 		String endpoint = endpoint(updateByQueryRequest.indices(), updateByQueryRequest.getDocTypes(), "_update_by_query");
 		Request request = new Request(HttpMethod.POST.name(), endpoint);
-		Params params = new Params(request).withRouting(updateByQueryRequest.getRouting())
-				.withPipeline(updateByQueryRequest.getPipeline()).withRefresh(updateByQueryRequest.isRefresh())
+		Params params = new Params(request)
+				.withRouting(updateByQueryRequest.getRouting())
+				.withPipeline(updateByQueryRequest.getPipeline())
+				.withRefresh(updateByQueryRequest.isRefresh())
 				.withTimeout(updateByQueryRequest.getTimeout())
 				.withWaitForActiveShards(updateByQueryRequest.getWaitForActiveShards())
 				.withRequestsPerSecond(updateByQueryRequest.getRequestsPerSecond())
```

Comment:
If you want to have this formatting, you need to add `// ` to the end of every line. Otherwise the formatter settings will set it back to the longer lines.

I would prefer to change the formatter, but that's the Spring Data style.

Reply:
done